logging/bin/deploy_logging_open.sh

```diff
@@ -40,28 +40,28 @@ log_notice "Deploying logging components to the [$LOG_NS] namespace [$(date)]"
 
 # Collect Kubernetes events as pseudo-log messages?
 
-log_info "STEP 1: Event Router"
+# log_info "STEP 1: Event Router"
 logging/bin/deploy_eventrouter.sh
```

Comment:
Is this being kept as a comment for people reviewing the code or are we still undecided if we want to write these messages out?  If it is the former, we can probably remove the function call.  And, the same applies to the rest of the "Step X" messages/comments below.